linux_os/guide/system/software/integrity/crypto/harden_sshd_crypto_policy/rule.yml

```diff
@@ -0,0 +1,20 @@
+documentation_complete: true
+
+title: 'Harden SSHD Crypto Policy'
+
+description: |-
+    Crypto Policies are means of enforcing certain cryptographic settings for selected applications including OpenSSH server.
+    The SSHD service is by default configured to enhance its configuration based on currently configured Crypto-Policy. However, in certain cases it is required to override the Crypto Policy specific to OpenSSH Server.
```

Comment:
`The SSHD service is by default configured to enhance its configuration based on currently configured Crypto-Policy.` I'm not sure I understand this thoroughly.
Mayby `The SSHD service is by default configured to extend its configuration ...` is what you mean?

Reply:
The configuration does not need to be extended necessarily in terms of being nlarged, I changed the wording to "modified".